xamarin/Xamarin.Forms
# [WPF] Fix issue setting initial ScrollView properties (VerticalScrollBarVisibility, etc)

### Description of Change ###

Fix issue setting initial ScrollView properties (VerticalScrollBarVisibility, etc) on WPF

### Issues Resolved ### 

- fixes #11709

### API Changes ###

 
 None

### Platforms Affected ### 

- WPF

### Behavioral/Visual Changes ###


None

### Before/After Screenshots ### 
![fix11709](https://user-images.githubusercontent.com/6755973/91182267-3db18000-e6ea-11ea-8c00-60a1f8c0a93c.gif)


### Testing Procedure ###
Launch Core Gallery and navigate to the issue 11709. If even being able to scroll in the ScrollView you cannot see the vertical scrollbar, the test has passed.

### PR Checklist ###
